incubator/node/templates/deployment.yaml

```diff
@@ -25,9 +39,8 @@ spec:
         - "/bin/bash"
         - "-ec"
         - |
-          DIR=$(echo $GIT_REPO | awk -F '/' '{print $NF }' | sed 's/.git$//g')
+          git checkout {{ .Values.revision }}
           chown -R bitnami:daemon /app 
-          cd /app/$DIR
           su  bitnami -c 'PATH=$PATH:/opt/bitnami/node/bin/ npm install' 
           su  bitnami -c 'PATH=$PATH:/opt/bitnami/node/bin/ npm start' 
```

Comment:
can we use the runAsUser security context instead so we can still run this as PID 1?